src/Nethermind/Ethereum.Test.Base/GeneralTestBase.cs

```diff
@@ -137,12 +137,20 @@ protected EthereumTestResult RunTest(GeneralStateTest test, ITxTracer txTracer)
 
             Block block = Build.A.Block.WithTransactions(test.Transaction).WithHeader(header).TestObject;
 
-            bool isValid = _txValidator.IsWellFormed(test.Transaction, spec) && IsValidBlock(block, specProvider);
+            bool blockIsValid = IsValidBlock(block, specProvider);
+            ValidationResult txIsValid = _txValidator.IsWellFormed(test.Transaction, spec);
```

Comment:
Calling BlockValidator should be enough and could replace those two lines:
  bool blockIsValid = IsValidBlock(block, specProvider);
            ValidationResult txIsValid = _txValidator.IsWellFormed(test.Transaction, spec);

Reply:
`IsValidBlock` will call `ValidateOrphanedBlock`, which does not validate tx. So i removed `IsValidBlock(block, specProvider)` instead, since it doesn't bring much value.